lib/iris/common/mixin.py

```diff
@@ -7,7 +7,6 @@
 
 from collections.abc import Mapping
 from functools import wraps
-import re
```

Comment:
`flake8` caught this... don't quite understand how it slipped though :thinking: i.e., the `re` module isn't used in `iris.common.mixin` any more